examples/go-V4Auth-example.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+    "fmt"
+    "time"
+    "bytes"
+    "encoding/json"
+    "net/http"
+    "io/ioutil"
+    "github.com/aws/aws-sdk-go/aws/credentials"
+    "github.com/aws/aws-sdk-go/aws/signer/v4"
+)
+
+func main() {
+    bucketName := "test-bucket"
+    d := time.Date(2017, 1, 19, 14, 45, 0, 0, time.UTC)
```

Comment:
Agreed! This keeps the response time of the example request constant as this script ages. Also, I have no favorite month, so January it is. 😄